spark/v3.4/spark-runtime/src/integration/java/org/apache/iceberg/spark/TestRoundTrip.java

```diff
@@ -172,6 +176,34 @@ public void showView() {
     sql("DROP VIEW %s", "test");
   }
 
+  @TestTemplate
+  public void testDecimalColumn() {
+    int rows = 100;
+    String tableName = tableName("decimal_table1");
```

Comment:
Shall we use the class level field `this.tableName` instead?
```
this.tableName = tableName("decimal_table1");
```
then we will have the  `@AfterEach` cleanup
 `sql("DROP TABLE IF EXISTS %s", tableName)`

Reply:
I think if we switch to class level field `this.tableName`, then we don't need the `TRUNCATE TABLE` below.